db/api/api.js

```diff
@@ -1,11 +1,18 @@
 const express = require("express");
-const { handle500, handle404 } = require("../api/controllers/handleErrors");
+const {
+  handle500,
+  handle404,
+  handleCustomErrors,
+} = require("../api/controllers/handleErrors");
 const { getCategories, getReviews } = require("./controllers/index");
 const app = express();
 
 app.get("/api/categories", getCategories);
 app.get("/api/reviews", getReviews);
-
+app.get("/api/reviews/:review_id", getReviews);
 app.all("*", handle404);
```

Comment:
careful with the name of this error handler `handle404` - this is a `bad path` error but that's not the only `404` status error you can have - `404` is when anything is not found in the database also